spark/src/main/scala/io/delta/tables/DeltaTable.scala

```diff
@@ -562,6 +562,119 @@ class DeltaTable private[tables](
           TableFeatureProtocolUtils.FEATURE_PROP_SUPPORTED))
     toDataset(sparkSession, alterTableCmd)
   }
+
+  /**
+   * Create a shallow clone of this DeltaTable to a new location.
```

Comment:
```suggestion
   *  Clone a DeltaTable to a given destination to mirror the existing table's data and metadata.
```

This will be more descriptive I think.